libraries/chain/database.cpp

```diff
@@ -2555,6 +2555,44 @@ namespace golos { namespace chain {
             }
         }
 
+        void database::process_worker_cashout() {
+            if (!has_hardfork(STEEMIT_HARDFORK_0_20__1013)) {
+                return;
+            }
+
+            const auto now = head_block_time();
+
+            const auto& wpo_idx = get_index<worker_proposal_index, by_next_cashout_time>();
+
+            for (auto wpo_itr = wpo_idx.begin(); wpo_itr != wpo_idx.end() && wpo_itr->next_cashout_time <= now; ++wpo_itr) {
```

Comment:
Worker proposal  is an idea to improve the blockchain. It can be discusses by different people.
It shouldn't contains information about payments for complete work.

Each proposal can contains several technical specification, which are made by technical specialist, and paying is made for technical specification not for proposal.

Move information about payments to technical specification.